bigday/settings.py

```diff
@@ -126,19 +126,25 @@
     os.path.join('bigday', 'static'),
 )
 
+# This is used in a few places where the names of the couple are used
+BRIDE_AND_GROOM = 'Bride and Groom'
+# base address for all emails
+DEFAULT_WEDDING_EMAIL = 'happilyeverafter@example.com'
 # the address your emails (save the dates/invites/etc.) will come from
-DEFAULT_WEDDING_FROM_EMAIL = 'You and Your Partner <happilyeverafter@example.com>'
+DEFAULT_WEDDING_FROM_EMAIL = BRIDE_AND_GROOM + ' ' + DEFAULT_WEDDING_EMAIL # change to 'address@domain.tld'
```

Comment:
Is the `<>` enclosure not necessary? (admittedly I forget how this is used)

Reply:
Good point, I'll correct shortly.